### PR TITLE
Enable remote debugging docker containers from vs code

### PR DESCRIPTION
**Pull request by mguidon**
## What do these changes do?

Adds configuration example for remote debugging a container [storage] from vs code. 

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] **Runs in the swarm**
- [ ] If you design a new module, add your user to .github/CODEOWNERS

### CODE REVIEW

File: services/docker-compose.devel.yml [hunk 1]
```diff
@@ -63,6 +63,10 @@ services:
     volumes:
       - ./storage:/devel/services/storage
       - ../packages:/devel/packages
+    environment:
+      - SC_BOOT_MODE=debug-ptvsd-disable
```
Comment (pcrespov) [inline services/docker-compose.devel.yml]: why the value assigned finishes with **disable** ??

File: services/docker-compose.devel.yml [hunk 2]
```diff
@@ -63,6 +63,10 @@ services:
...
+    ports:
+      - "3000:3000"
```
Comment (pcrespov) [inline services/docker-compose.devel.yml]: Do we have any kind of criteria to assign port numbers?

File: .vscode-template/launch.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      "name": "Python: Remote Attach storage",
+      "type": "python",
+      "request": "attach",
+      "port": 3000,
+      "host": "127.0.0.1",
+      "pathMappings": [
+        {
+          "localRoot": "${workspaceFolder}/services/storage",
+          "remoteRoot": "/devel/services/storage"
+        }
```
Comment (pcrespov) [inline .vscode-template/launch.json]: So this is ONLY for development mode, right?

File: services/docker-compose.devel.yml (hunk 1 above)
Comment (mguidon) [reply services/docker-compose.devel.yml]: to disable it

File: services/docker-compose.devel.yml (hunk 2 above)
Comment (mguidon) [reply services/docker-compose.devel.yml]: no, but I guess we can define some rules

File: .vscode-template/launch.json (hunk 3 above)
Comment (mguidon) [reply .vscode-template/launch.json]: this example yes

**Pull request closed without merging**